client/apis/volumesnapshot/v1/types.go

```diff
@@ -29,7 +29,7 @@ import (
 // VolumeSnapshot is a user's request for either creating a point-in-time
 // snapshot of a persistent volume, or binding to a pre-existing snapshot.
 // +kubebuilder:object:root=true
-// +kubebuilder:resource:scope=Namespaced
+// +kubebuilder:resource:scope=Namespaced,shortName=vs
```

Comment:
Does this not require a plural?

Reply:
I'm not sure. It felt funny to type `vss`, but I can certainly add it if you like. I will note that the short name for stateful sets, `sts`, doesn't have a plural

Reply:
I wonder if there's a convention for that.  If StatefulSet does not have a plural, we are probably fine.

Reply:
I couldn't find any published conventions. I reached out to SIG-API-Machinery, SIG-Arch, and SIG-Usability via Slack, but didn't get a response